import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreService.java

```diff
@@ -64,6 +70,9 @@ protected void store(
       String transferSyntax = presentationContext.getTransferSyntax();
       String remoteAeTitle = association.getCallingAET();
 
+      log.info("Received C-STORE for association {}, SOP class {}, TS {}, remote AE {}",
```

Comment:
Let's spell out transfer syntax since the logs are user visible. Also should we have logging like this for the other services like CFIND, MOVE etc?

Reply:
It may be actually best to remove this line, since it's somewhat legacy. Dcm4che already logs it all, in a good format.

(last message is from this line, rest is dcm4che standard)

```
DEBUG org.dcm4che3.net.Association  - IMPORTADAPTER<-STORESCU(1): enter state: Sta6 - Association established and ready for data transfer
INFO  org.dcm4che3.net.Dimse  - IMPORTADAPTER<-STORESCU(1) >> 1:C-STORE-RQ[pcid=5, prior=0
  cuid=1.2.840.10008.5.1.4.1.1.1.2 - Digital Mammography X-Ray Image Storage - For Presentation
  iuid=1.3.6.1.4.1.5962.1.178.1174395404.11033.0 - ?
  tsuid=1.2.840.10008.1.2.1 - Explicit VR Little Endian
DEBUG org.dcm4che3.net.Dimse  - Command:
(0000,0002) UI [1.2.840.10008.5.1.4.1.1.1.2] AffectedSOPClassUID
(0000,0100) US [1] CommandField
(0000,0110) US [1] MessageID
(0000,0700) US [0] Priority
(0000,0800) US [0] CommandDataSetType
(0000,1000) UI [1.3.6.1.4.1.5962.1.178.1174395404.11033.0] AffectedSOPInstance

INFO  com.google.cloud.healthcare.imaging.dicomadapter.CStoreService  - Received C-STORE for association IMPORTADAPTER<-STORESCU(1), SOP class 1.2.840.10008.5.1.4.1.1.1.2, TS 1.2.840.10008.1.2.1, remote AE STORESCU
```

Reply:
I'm good with removing this line.